source/jsonnet/england-wales/ccs_household.jsonnet

```diff
@@ -164,7 +164,7 @@ function(region_code, census_month_year_date) {
       groups: [
         {
           id: 'accommodation-group',
-          title: '',
+          title: 'Accommodation group',
```

Comment:
If these aren't displayed, then we shouldn't define them.

Reply:
They are required properties at the moment, so I can't remove them unless I change validator, and can't leave them empty due to the new validation rules.

Reply:
We should consider whether we should remove keys that are not needed by runner. Obviously this is required in validator but not needed in runner, so defining keys that don't get used seems wrong.